### ISSUE TITLE
Publish on PyPi

### ISSUE THREAD

**Issue by roquelopez** [ivanp1994/PyMart#4]
Could you please publish the current version (the one that fixes the issues with dataclass) on PyPi? PyPi contains an old version (August 2023). Thanks!

**Comment by roquelopez** [comment]
Are there any plans to publish the newest version on PyPi? Thanks!

**Comment by ivanp1994** [comment]
Yep, I'm just away from home for a week or so. I'll do it when I get home
ASAP

**Comment by roquelopez** [comment]
Thanks!

**Comment by keviny2** [comment]
any movement on this?

**Comment by ivanp1994** [comment]
Yes, will do that later today.

čet, 2. sij 2025. 16:19 Kevin Yang ***@***.***> je napisao:

**Comment by hasanur-rahman079** [comment]
any update on this?

**Comment by keviny2** [comment]
bump

**Comment by BairdMcIlwraith** [comment]
Would be great if this could be done, having problems with conda and github. Thanks!
